Index: RenderSystems/GL/src/GLX/OgreGLXWindow.cpp
===================================================================
--- RenderSystems/GL/src/GLX/OgreGLXWindow.cpp	(revision 8673)
+++ RenderSystems/GL/src/GLX/OgreGLXWindow.cpp	(working copy)
@@ -543,6 +543,8 @@
 				mWidth = width;
 				mHeight = height;
 				
+				XResizeWindow(mGLSupport->getXDisplay(), mWindow, width, height);
+
 				for (ViewportList::iterator it = mViewportList.begin();	it != mViewportList.end(); ++it)
 					(*it).second->_updateDimensions();
 			}
